### ISSUE TITLE
Exception

### ISSUE THREAD

**Issue by kinv** [ddurdle/Python-GoogleDrive-VideoStream#127]
Hi, Have multiple TD's mounted. All but one works fine.

I have tried:

Deleting the task and retrying,
Re-auth and re try the task,
Pulling latest.
Trying again on a fresh install,
Verifying I still have access to the TD.

The log tells me 

```
Line 21 : exception 
Line 22 : exception making call to http://173.249.48.125:9988/default.py?mode=buildstrm&logfile=/tmp/tv1&host=https://strm.kinv.pw&force=False&remove_ext=True&resolution=True&helperfiles=True&append=&catalog=False&strm_path=/u01/STRM&content_type=video&folder=0ACjQTLTJT4IpUk9PVA&filename=None&original=True&transcode=False&skip=True&title=[Team%20Drive%20-%20zd_tv1]&username=MOB&encfs=False&epath=&dpath=&change_token=952920, giving up. 
Line 23 : making call to API 
Line 24 : runtime & status updated 
Line 25 : time to run job #10 runtime 0 test 1551715433cmd<ip>:9988/default.py?mode=buildstrm&logfile=/tmp/tv1&host=<domain>&force=False&remove_ext=True&resolution=True&helperfiles=True&append=&catalog=False&strm_path=/u01/STRM&content_type=video&folder=>foldername>&filename=None&original=True&transcode=False&skip=True&title=[Team Drive - <ID>]&username=<user>&encfs=False&epath=&dpath= 
```
note <> I have changed for privacy reasons.

Log file for the specif scheduled task outputs

` 
folder id       folder title    file id file title      show title (tv) season (tv)     episode (tv)    title (movie)   year (movie)    video resolution    file checksum    file size

`

No Idea whats causing this as the other tasks perform no problem. This was working till a few days ago as well.

**Comment by ddurdle** [comment]
are you using username/password for the web ui?

For the URL in the job, have you tried opening it remotely from your computer to see what results (hang, error, or prompt for username/password)?

**Comment by kinv** [comment]
No password.

When I click on the job in the UI it says the page didn't send any data. Even on the jobs that are working.

**Comment by ddurdle** [comment]
Make sure to check the schedule.log for the error.  I'm 100% certain you have some filesystem error in the STRM path or some other kind of other i/o error that will reveal itself when the job runs in either the server.log or the output from the process.

**Comment by kinv** [comment]
Tracked the error, forgot to look in one of the log files. "File name too long". Whats the filename limit and can you increase it?

**Comment by ddurdle** [comment]
File name for a STRM file?

**Comment by kinv** [comment]
yeah, think TVMH pmed you a log file for me. He uses same TD's and for him the file was skipped and the task kept running for me it exits with exception.

**Comment by ddurdle** [comment]
I think his issue was with a SRT file.  I can try making this more redundant to skip the error and continue.  I'll investigate today.

**Comment by kinv** [comment]
Hi, Tried with download srt set to false, same error.

**Comment by ddurdle** [comment]
linked to https://github.com/ddurdle/Python-GoogleDrive-VideoStream/issues/84

**Comment by kinv** [comment]
Had any luck with this?

**Comment by Drondemovies** [comment]
put a www. in front of your url
